lib/__tests__/helpers/createMockManager.js

```diff
@@ -7,6 +7,7 @@ module.exports = function() {
     openMenu: jest.fn(),
     handleKeyDown: jest.fn(),
     handleClick: jest.fn(),
+    handleBlur: jest.fn(),
```

Comment:
You can just leave it in here. Strange that these tests have not failed for me, or for CI, though.